deno_dist/middleware/secure-headers/index.ts

```diff
@@ -126,6 +126,35 @@ export const NONCE: ContentSecurityPolicyOptionHandler = (ctx) => {
   return `'nonce-${nonce}'`
 }
 
+/**
+ * Secure Headers Middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/secure-headers}
+ *
+ * @param {Partial<SecureHeadersOptions>} [customOptions] - The options for the secure headers middleware.
```

Comment:
Remove `Partial` since #2713 was merged.

Reply:
Thanks! Resolved and squashed to first commit.